### PR TITLE
iscomplex

### LINKED ISSUE #8840: iscomplex

### PR DESCRIPTION
**Pull request by Mukhriddin19980901**
Fixes #8840

### CODE REVIEW

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ZoeCD** [comment]
Hi! It seems there are only a few errors to fix before emerging this PR. You can see more information in the `Details` section of the CI tests 🙂

**Comment by Mukhriddin19980901** [comment]
> Hi! It seems there are only a few errors to fix before emerging this PR. You can see more information in the `Details` section of the CI tests 🙂

Thanks!I'll try to fix it.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

File: ivy/functional/frontends/numpy/logic/truth_value_testing.py [hunk 1]
```diff
@@ -40,3 +40,11 @@ def isscalar(element):
...
         or isinstance(element, float)
     )
+
+@to_ivy_arrays_and_back   
+def iscomplex(element):
+    ax = ivy.asarray(element)
+    if issubclass(ax.dtype.type, ivy.complexfloating):
+        return ax.imag != 0
+    res = ivy.zeros(ax.shape, bool)
+    return res[()]
```
Comment (ZoeCD) [inline ivy/functional/frontends/numpy/logic/truth_value_testing.py:50]: There are some lint errors. Did you install the pre-commit hook? Also, I'll appreciate it if you tag me once you finish editing your PR; so I know it's ready. Thanks! 😊

**Review by ZoeCD — CHANGES_REQUESTED** [review]

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

File: ivy/functional/frontends/numpy/logic/truth_value_testing.py (hunk 1 above)
Comment (Mukhriddin19980901) [reply ivy/functional/frontends/numpy/logic/truth_value_testing.py:50]: Hi @ZoeCD ,
I have installed the pre-commit hook but I don't know if it's a suitable version.Can please check  ,once again?

**Comment by Mukhriddin19980901** [comment]
#9073

**Comment by Mukhriddin19980901** [comment]
Fixes #9073

File: ivy/functional/frontends/numpy/logic/truth_value_testing.py (hunk 1 above)
Comment (ZoeCD) [reply ivy/functional/frontends/numpy/logic/truth_value_testing.py:50]: Just don't forget to leave a line at the end of the file

**Review by ZoeCD — CHANGES_REQUESTED** [review]
There are some conflicting files. Can you review that, please? 😊

File: ivy/functional/frontends/numpy/logic/truth_value_testing.py (hunk 1 above)
Comment (Mukhriddin19980901) [reply ivy/functional/frontends/numpy/logic/truth_value_testing.py:50]: Ok.🙂.Thank you for reviews.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**